testing/bdd/features/validation/applied-processing.feature

```diff
@@ -0,0 +1,30 @@
+@validation @syntax @metadata
+Feature: Applied processing element constrainst
+  Attributes `process` and `generatedBy` are mandatory.
+
+  # SPEC-CONFORMANCE : R31 R42 R43
+  Scenario: Invalid applied processing attributes
+    Given an xml file <xml_file>
+    When trace element has process attribute <process>
+    And trace element has generatedBy attribute <generated_by>
+    And trace element has sourceId attribute <source_id>
```

Comment:
On the above three lines, shouldn't it be `applied processing element` in place of `trace element`?